src/assets/scripts/client/ui/TrafficRateController.js

```diff
@@ -176,6 +182,23 @@ export default class TrafficRateController {
         this._buildDialogBody();
     }
 
+    /**
+     * Resets the traffic dialog and spawn rates to default values
+     *
+     * @for TrafficRateController
+     * @method onRateReset
+     */
+    onFormReset() {
+        const airport = AirportController.airport_get();
+        this._wind = { speed: airport.defaultWind.speed, angle: Math.round(radiansToDegrees(airport.defaultWind.angle)) };
+
+        this._eventBus.trigger(EVENT.WIND_CHANGE, this._wind);
+        _forEach(SpawnPatternCollection.spawnPatternModels, (spawnPattern) => {
```

Comment:
I'd encourage moving this work to the `SpawnPatternCollection` itself and exposing a method `.resetSpawnPatternRates()` or something.

Generally, when dealing with a collection of items, we try to have the collection do the work and simply expose methods for consumers to call. That keeps everything centralized and makes it easier to track down where things are changing.

Additionally, when you do move this to the collection, feel free to use the built-in foreach instead of lodash. This app went through a heavy refactor when the built-ins weren't yet built-in, so we used lodash. We don't need that anymore (for this anyway).